bs/src/tablecloth.mli

```diff
@@ -1527,41 +1527,345 @@ module Result : sig
     -> unit
 end
 
+(**
+  This module implements the `Option` type, which has a variant for 
+  valid values (`'Some`), and one for invalid values (`'None`).
+*)
+
 module Option : sig
+  (**
+    `type` is the type constructor for an `Option` type. You specify
+    the type of the `Some` variant.
```

Comment:
I dont understand what this says.

Reply:
OK; I will get rid of it entirely, since we want people to work with `option` instead.